### ISSUE TITLE
Theme with Scanlines

### ISSUE THREAD

**Issue by AndreSanctus** [virgildu68/es-theme-carbon-v2#1]
Hi. I am using Batocera with the default theme Carbon, and it has an option in the theme configuration which I love: Theme Shader: Scanlines/CRT/None.

I love to use scanlines in the theme however, after I installed this Carbon theme of yours, on the:
Main menu->System settings->Updates & Downloads->Themes

I lost the option to add scanlines in the Theme Configuration menu.

Do you have a solution?

**Comment by fabricecaruso** [comment]
Yes : Don't try to install this theme on non-development Batocera ES version.
Just wait for the next batocera full build instead - It will include this theme & the proper ES version.

My repository is under permanent development & implement new features that run only aligned to the development version.
